test/e2e/e2eutil/wait_util.go

```diff
@@ -0,0 +1,158 @@
+// Copyright 2016 The etcd-operator Authors
```

Comment:
I'll change the licenses here but we'll need to update everywhere else sometime later.